### DISCUSSION TITLE
Why create the same repo over again?

### DISCUSSION THREAD

**Discussion by rfalanga** [skills/.github#66] [Q&A]
Although I've used GitHub for a few years, I just learned of your [Skills](https://skills.github.com) website, so I thought I'd go through it to fill in gaps in my understanding.

I went through the *Introduction to GitHub* first module, which had me create the introduction-to-github repo.

Then, I went to module two, on writing Markdown. The first thing it wants me to do is create the introduction-to-github repo. Again. Huh? Do I use the one created in module 1? Do I delete the introduction-to-github repo? There is no `Use this template` button. I'm lost as to what I'm supposed to do in order to have a pull request created. I don't understand why I can't continue to use the introduction-to-github repo I did in module 1. 

Please clarify.

**Comment by sinsukehlab** [comment]
In my opinion, it is not a good practice to create a repository with the same name as the template.
The "Create a new repository" image is misleading.

**Reply by sinsukehlab** [reply]
I have created an issue from this discussion.
Cf. skills/template-template#21

**Reply by rfalanga** [reply]
Thank you, @sinsukehlab
